Work-around rhbz#1517657 v8 bug, we need to create the
libv8_libplatform.so link in CWD and pass the linker flags
down to gcc through $(SHLIB_LINK).

diff --git a/Makefile b/Makefile
index f410301..2ed035b 100644
--- a/Makefile
+++ b/Makefile
@@ -43,7 +43,7 @@ ifndef DISABLE_DIALECT
 REGRESS += dialect
 endif
 
-SHLIB_LINK += -lv8
+SHLIB_LINK += -lv8 $(RPM_HACK_LDFLAGS)
 ifdef V8_OUTDIR
 SHLIB_LINK += -L$(V8_OUTDIR)
 else
